### PR TITLE
Fixed failing tests for Drupal 9 support

### LINKED ISSUE #42: Drupal 9 test failing: Contents Block (Drupal\Tests\localgov_guides\Functional\ContentsBlock)
```
Contents Block (Drupal\Tests\localgov_guides\Functional\ContentsBlock)
 ✔ Contents block visibility
 ✘ Content list block
   ┐
   ├ TypeError: Argument 2 passed to PHPUnit\Framework\Assert::assertContains() must be iterable, string given, called in /app/web/modules/contrib/localgov_guides/tests/src/Functional/ContentsBlockTest.php on line 114
   │
   ╵ /app/vendor/phpunit/phpunit/src/Framework/Assert.php:169
   ╵ /app/web/modules/contrib/localgov_guides/tests/src/Functional/ContentsBlockTest.php:114
   ╵ /app/vendor/phpunit/phpunit/src/Framework/TestResult.php:722
   ┴

### PR DESCRIPTION
**Pull request by stephen-cox**
Fixes failing PHPUnit tests when running Drupal 9. Includes fixes for code deprecation warnings as well.

Closes #42

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Test LocalGovDrupal Guides
+name: Test LocalGov Drupal
```
Comment (Adnan-cds) [inline .github/workflows/test.yml:1]: Does the name of the test really need to change?

**Review by Adnan-cds — COMMENTED** [review]
Love the changes.  Particularly the *void* return type :)  I have request just one clarification.  Otherwise looks great.

File: .github/workflows/test.yml (hunk 1 above)
Comment (ekes) [reply .github/workflows/test.yml:1]: Guess this should be Test LocalGov Drupal Guides?

@stephen-cox Do you want to confirm and I'll edit and merg.

Comment (stephen-cox) [reply .github/workflows/test.yml:1]: @ekes It should indeed be 'Test LocalGov Drupal Guides'. Happy for you to fix and merge. I'll do it later if you don't get around to it.

Comment (ekes) [reply .github/workflows/test.yml]: Updated it in the github editor :)

**Pull request merged**